### PR TITLE
Source File Secure: bump source-file version 0.2.38 -> 0.3.7

### PR DESCRIPTION
**Pull request by grubberr**
## What

I want to re-publish `source-file` and `source-file-secure` because 
`source-file-secure` is based on top of out-dated version of `source-file:0.2.38`

### CODE REVIEW

**Review by artem1205 — APPROVED** [review]

**Comment by grubberr** [comment]
already done here  https://github.com/airbytehq/airbyte/commit/59903d04b9ad9f33bc0f32eaa4a7bb0a4d336312

**Review by evantahler — APPROVED** [review]

**Comment by bnchrch** [comment]
Hey @grubberr! Just looking at this now.

Im sorry we missed the version at the top of the dockerfile!

I think youre going the correct way and updating both to 0.3.7 to accomodate.

File: airbyte-integrations/connectors/source-file/metadata.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ data:
   connectorSubtype: file
   connectorType: source
   definitionId: 778daa7c-feaf-4db6-96f3-70fd645acc77
-  dockerImageTag: 0.3.6
+  dockerImageTag: 0.3.7
```
Comment (bnchrch) [inline airbyte-integrations/connectors/source-file/metadata.yaml:8]: We also need to update the cloud pinned version

https://github.com/airbytehq/airbyte/pull/26131/files#diff-580b4a25c9d6e8ce3172870d4a6eb1064426fc4c557c69b90361902bc66d39dfL17

**Review by bazarnov — APPROVED** [review]

**Review by bnchrch — APPROVED** [review]
LGTM! Just make sure to run /publish for both source-file and source-file-secure

**Comment by bnchrch** [comment]
Hey @grubberr 

Ok so I noticed that the source-file-secure is not publishing because its spec tests are failing
https://scans.gradle.com/s/pfciulrg3tgr2/console-log?task=:airbyte-integrations:connectors:source-file-secure:connectorAcceptanceTest
image

To resolve this I think we have two choices
1. Fix the spec in the source-file-secure


**Wait, nvm, I see you just did that! Woohoo!**

**Comment by bnchrch** [comment]
Next onto the failing definitions.

That was my fault. Weve removed the auto bump feature. And /publish still references it.

Ill remove that ASAP

**Pull request merged**